indico/modules/events/tracks/models/tracks.py

```diff
@@ -108,3 +108,11 @@ def __repr__(self):
 
     def can_delete(self, user):
         return self.event_new.can_manage(user) and not self.abstracts_accepted
+
+    def is_user_reviewer(self, user):
```

Comment:
I agree `can_convene` sounds awful. Anyway, I'm using your naming since it is more consistent with what we have.